docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -204,6 +211,15 @@ We can send an http request directly to the Broker with correct CloudEvent heade
 
       exit
       ```
+
+**Note** : For v0.18 or above, use `http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default` as the address insteed of `default-broker.knative-samples.svc.cluster.local` . Your can view the addree by :
+
+```shell
+$ kubectl -n knative-samples get broker
+NAME      URL                                                                                AGE   READY   REASON
+default   http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default   7m    True
+```
+
```

Comment:
I think we should just fix the example since all samples are versioned with the documentation.

/cc abrennan89

Reply:
100% correct @pierDipi